doc/dynamic-environments/git-environments.mkd

```diff
@@ -45,6 +45,10 @@ future r10k runs. Finally, if there are directories that do not match existing
 branches, r10k will assume that the branches for those environments were delete
 and will remove those environments.
 
+Each repository that r10k needs to communicate with will require a [Deploy Key]
```

Comment:
Deploy key doesn't need to be capitalized, but that's minor.  More importantly, this documentation only applies to GitHub.  It may be worth changing the wording to something like:

```markdown
r10k will need to be be able to authenticate with each repository. Most Git systems support authentication with SSH keys. GitHub calls them [deploy keys](https://developer.github.com/guides/managing-deploy-keys/#deploy-keys). BitBucket calls them [deployment keys](https://confluence.atlassian.com/display/BITBUCKET/Use+deployment+keys). Stash calls them [SSH access keys](https://confluence.atlassian.com/display/STASH/SSH+access+keys+for+system+use).
```